src/pages/home/home.scss

```diff
@@ -5,6 +5,13 @@ page-home {
       background: $feather;
     }
 
+    &.content-md {
```

Comment:
`*-md` mean it only works for Android (or web for devs). Is that ok?

Reply:
Yes, this fix is just for android devices